src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/config/GivensBuilder.java

```diff
@@ -133,9 +134,9 @@ public GivensBuilder iCreateATaskWithCaseId(String caseId, String taskType) {
         return this;
     }
 
-    public List<CamundaTask> iRetrieveATaskWithProcessVariableFilter(String key, String value) {
+    public List<CamundaTask> iRetrieveATasksWithProcessVariableFilter(String key, String value, String taskType) {
```

Comment:
Maybe a list of key/values would give more flexibility here for further additions?